src/scheduler/job_info.c

```diff
@@ -3026,7 +3026,8 @@ find_jobs_to_preempt(status *policy, resource_resv *hjob, server_info *sinfo, in
 	int j = 0;
 	int has_lower_jobs = 0;	/* there are jobs of a lower preempt priority */
 	int prev_prio;		/* jinfo's preempt field before simulation */
-	server_info *nsinfo;
+	server_info *nsinfo = NULL;
+	status *npolicy = NULL;
```

Comment:
No need to initialize these.  They get set before they are used.

Reply:
All of the local pointers besides nsinfo in the function seem to be initialized, I checked a few of them and they seem to be getting set before use as well. Do you still want me to revert it?

Reply:
Please remove the initialization.  Feel free to fix other pointers that get set before they are used as well.  Now that we have gcc -Wall -Werror set, the compiler will bail if we use an uninitialized variable.  If we always initialize them all, gcc won't alert us and we'll have a bug.

Reply:
I see, alright I'll remove them, thanks.